server/initializer/initializer.go

```diff
@@ -0,0 +1,91 @@
+package initializer
+
+import (
+	"github.com/MacroPower/macropower-analytics-panel/server/cacher"
+	"github.com/MacroPower/macropower-analytics-panel/server/payload"
+	"github.com/MacroPower/macropower-analytics-panel/server/worker"
+	"github.com/go-kit/kit/log"
+	"github.com/google/uuid"
+	"time"
+)
+
+func InitializeMetricsForDashboards(api worker.Client, logger log.Logger, cache *cacher.Cacher) {
+	dashboards, hasErrored := api.GetDashboards()
+	if hasErrored {
+		return
+	}
+
+	for _, dashboard := range dashboards {
+		payloadData := createDashboardPayload(dashboard.Uid, dashboard.Title, api.GrafanaUrl)
+
+		payload.ProcessPayload(cache, payloadData, logger)
+	}
+}
+
+func createDashboardPayload(dashboardUid string, dashboardName string, grafanaUrl string) payload.Payload {
```

Comment:
misleading name, as it creates panel, not dashboard?

Reply:
It creates payload to emulate dashboard visit.